README.md

```diff
@@ -152,30 +152,9 @@ Documentation:	https://docs.styra.com/regal/rules/style/prefer-snake-case
 > will likely generate a lot of violations. You can do this by passing the `--disable-category style` flag to
 > `regal lint`.
 
-### GitHub Actions
+### Using Regal in CI/CD Workflows
```

Comment:
"Using Regal in your build pipeline" sounds a little less techno-babbly, perhaps?